go/cmd/dolt/dolt.go

```diff
@@ -470,7 +528,7 @@ The sql subcommand is currently the only command that uses these flags. All othe
 	}
 
 	var cliCtx cli.CliContext = nil
-	if initCliContext {
+	if initCliContext(remainingArgs[0]) {
```

Comment:
Assign `remainingArgs[0]` to a named variable, so it's clear that this is the command name.

Reply:
Done